lib/recipe.rb

```diff
@@ -289,8 +399,8 @@
         <string>wheel</string>
         <key>KeepAlive</key>
         <dict>
-	              <key>SuccessfulExit</key>
-	              <true/>
+                      <key>SuccessfulExit</key>
```

Comment:
I don't think this was supposed to happen. BSD init scripts are picky about tabs and spaces.